From 135f4233266443a4546bdf8217f145a0f7bbb90d Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@gmail.com>
Date: Thu, 7 Sep 2017 22:25:48 +0200
Subject: [PATCH] eina_debug: add missing header for big endian build

The build fail for big endian system due to missing eina_swap32
definition.
Add "eina_cpu.h" include where eina_swap32 prototype is defined.

Patch provided by sharkcz in [1].

Uptream status: pending

[1] https://phab.enlightenment.org/D5158

Signed-off-by: Romain Naour <romain.naour@gmail.com>
---
 src/lib/eina/eina_debug.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lib/eina/eina_debug.c b/src/lib/eina/eina_debug.c
index ca1bdad..468ca43 100644
--- a/src/lib/eina/eina_debug.c
+++ b/src/lib/eina/eina_debug.c
@@ -60,6 +60,7 @@
 #endif
 
 #include "eina_debug.h"
+#include "eina_cpu.h"
 #include "eina_types.h"
 #include "eina_list.h"
 #include "eina_mempool.h"
-- 
2.9.5

